janus_core/helpers/mlip_calculators.py

```diff
@@ -217,6 +217,36 @@ def choose_calculator(
         kwargs.setdefault("sevennet_config", None)
         calculator = SevenNetCalculator(model=model, device=device, **kwargs)
 
+    elif arch == "orb":
+        __version__ = "0.3"
+        from orb_models.forcefield.calculator import ORBCalculator
+        from orb_models.forcefield.graph_regressor import GraphRegressor
+        import orb_models.forcefield.pretrained as orb_ff
+
+        if isinstance(model_path, str):
+            match model_path:
+                case "orb-v1":
+                    model = orb_ff.orb_v1()
+                case "orb-mptraj-only-v1":
+                    model = orb_ff.orb_v1_mptraj_only()
+                case "orb-d3-v1":
+                    model = orb_ff.orb_d3_v1()
+                case "orb-d3-xs-v1":
+                    model = orb_ff.orb_d3_xs_v1()
+                case "orb-d3-sm-v1":
+                    model = orb_ff.orb_d3_sm_v1()
+                case _:
+                    raise ValueError(
+                        "Please specify `model_path`, as there is no "
+                        f"default model for {arch}"
+                    )
```

Comment:
Should be split into multiple lines, but this is the gist
```suggestion
            model = getattr(orb_ff, model_path.sub("-", "_"), None)()
            if model is None:
                    raise ValueError(
                        "Please specify `model_path`, as there is no "
                        f"default model for {arch}"
                    )
```

Reply:
would be nice but orb-mptraj does not match the pattern.

Reply:
Don't we choose what "model_path" is? You could also easily special case that. Here, it's not clear that that *is* different at first glance.

Reply:
no really I user the canonical names from their website... which i suspect is what people will expect to call them

Reply:
this one is different buy maybe we can import it as the write pattern and then use your abstraction?

case "orb-mptraj-only-v1":
                    model = orb_ff.orb_v1_mptraj_only()